### DISCUSSION TITLE
Zoom feature for iOS

### DISCUSSION THREAD

**Discussion by nghiavt1302** [mebjas/html5-qrcode#673] [Ideas]
Can you please make the zoom slider function for iOS devices? Or at least the function to set the default zoom level?
Many thanks.

**Comment by ROBERT-MCDOWELL** [comment]
zoom features depend from the phone device itself and its ability to accept its webcam to use zoom from javascript, it has nothing to do with html5-qrcode project

**Reply by jmldsilva** [reply]
Hello, any updates for this topic?

**Comment by JeremyShafer** [comment]
I absolutely love this project... and it works great on my Android phone.  But, sadly, the absence of a zoom slider on IOS devices is proving to be a challenge for me.  I understand that Apple just might not play nice here... but it would be so great to have this addressed if at all possible.  

I am open to creative workarounds and alternatives too.  I am fishing for ideas.

-Shafer

**Comment by ROBERT-MCDOWELL** [comment]
@JeremyShafer 
again, this is not a html5-qrcode issue but iOS issue... a web browser is limited by the javascript API whic html5-qrcode use.
if there is a bug or non standard behavior on iOS and javascript API in browsers so you should contact Apple
